source/docs/index.html

```diff
@@ -224,13 +236,14 @@ <h3 class="panel-title-small">Welcome to our new office</h3>
 <script src="../../bower_components/holderjs/holder.js"></script>
 <script src="../../bower_components/highlightjs/highlight.pack.js"></script>
 <script src="../../bower_components/angular-translate/angular-translate.js"></script>
+<script src="../../bower_components/angular-ui-notification/dist/angular-ui-notification.min.js"></script>
```

Comment:
I usually choose the un-minified version in here to make the source more readable, if necessary. It's fine though.